debian/controlX/control.xenial

```diff
@@ -20,22 +20,21 @@ Depends:
  ${misc:Depends}
  , apparmor
  , apparmor-utils
+ , tor
  , haveged
  , iptables
- , python (>= 2.7)
- , python-cryptography
- , python-cyclone
- , python-gnupg
- , python-openssl
- , python-scrypt
- , python-storm
- , python-transaction
- , python-twisted-core
- , python-zope.component
- , python-zope.event
- , python-zope.interface
-Recommends:
- tor
+ , python (>=2.7)
+ , python-pip
+ , python-cryptography (>=1.2.3)
+ , python-cyclone (>=1.1)
+ , python-gnupg (>=0.3.8)
+ , python-openssl (>=0.15.1)
+ , python-scrypt (>=0.6.1)
+ , python-storm (>=0.19)
+ , python-transaction (>=1.4.3)
+ , python-twisted-core (>=16.0.0-1)
+ , python-txsocksx (>=1.15.0.2-1)
+ , ${python:Depends}
```

Comment:
@evilaliv3 `{python:Depends}` has snuck in again. The deb package as is installs without a hitch though.